configure.ac

```diff
@@ -390,6 +390,8 @@ AC_CONFIG_FILES([check.pc
                  Makefile
                  checkmk/Makefile
 		 doc/Makefile
+    doc/man/Makefile
```

Comment:
The spacing did not line up before, though it lines up less now. If the spacing will not line up, could there be an additional commit that changes the spacing to spaces so it lines up?

Reply:
Absolutely, I'll make the change. I didn't want to format the other lines and replace the git blame data, but I'm not sure why I didn't then just match the existing format, sorry about that.

Reply:
I'll wait to see a commit in this PR, then will resolve this comment

Reply:
Done!